index.html

```diff
@@ -16,7 +16,7 @@
         <nav>
             <div class="info">
                 <a href="#" class="info-text">Ali Shah</a>
```

Comment:
- [ ] Great job so far @Team. After using [this aditus tool](https://www.aditus.io/button-contrast-checker/) to check the color contrast of your project, 3 of the elements including this text `Ali Shah` was failing so kindly use the tool to check and fix any contrast issues that might be present in your project :100: